src/modules/stream_preview/Preview.jsx

```diff
@@ -0,0 +1,18 @@
+import React, {useEffect, useState} from 'react';
+import styles from './Preview.module.css';
+
+function createPreviewImage(user) {
+  const dateTime = new Date().toISOString(); // cache prevention
+  return `https://static-cdn.jtvnw.net/previews-ttv/live_user_${user}-440x248.jpg?dateTime=${dateTime}`;
```

Comment:
can we get this from react instead? I believe there exists a stream object in the parent chain

Reply:
couldn't find a prop with this data